locale/en/manageiq.po

```diff
@@ -10,8 +10,8 @@ msgstr ""
 "Report-Msgid-Bugs-To: \n"
 "PO-Revision-Date: 2022-02-08 11:07-0500\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Language-Team: English\n"
+"Language: en\n"
```

Comment:
These values were actually being merged in the json file https://github.com/ManageIQ/manageiq-ui-classic/pull/8204 by the oracle cloud provider.  It seemed best to set these values here and not rely on an outside provider to set it.

Reply:
I'm not sure.  We don't set Plural-Forms correctly there `"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"` which is the cause of all of these problems in various po files if the translation po file is missing a correct value.

Reply:
For plural forms, those have to be changed for each language, so I guess the same argument could be made for all of these fields including Language. Thanks 👍

Reply:
Maybe we want a spec that shows all of the headers updated appropriately, but definnitely not for this PR